src/pages/trade/ui/summary.tsx

```diff
@@ -48,16 +48,18 @@ export const Summary = () => {
   const { quoteAsset } = usePathToMetadata();
 
   const change24h = data && {
-    positive: data.price >= data.price,
-    change: round(data.price - data.price, 4),
-    percent: round(Math.abs(((data.price - data.price) / data.price) * 100), 2),
+    positive: data.price >= data.price_then,
+    change: round(data.price - data.price_then, 4),
+    percent: !data.price
+      ? '0'
+      : round(Math.abs(((data.price - data.price_then) / data.price_then) * 100), 2),
   };
 
   if (error) {
     return (
-      <SummaryCard title='Error'>
+      <SummaryCard title=''>
```

Comment:
The Error title was a bit redundant. And we'll need to update this to the error design anyway in figma later. 

<img width="575" alt="Screenshot 2024-11-18 at 7 56 04 PM" src="https://github.com/user-attachments/assets/58db934d-ecab-40b8-b0d9-204255e19394">